scripts/e2e_setup_cluster.sh

```diff
@@ -142,18 +153,18 @@ retry kubectl create -f "${CNIS_DAEMONSET_URL}"
 retry kubectl -n kube-system wait --for=condition=ready -l name="${CNIS_NAME}" pod --timeout=300s
 echo "## install NRI"
 retry kubectl create -f "${root}/deployments/auth.yaml"
-retry kubectl create -f "${root}/deployments/server.yaml"
+retry kubectl create -f "${root}/deployments/server_huge.yaml"
```

Comment:
Maybe we can reuse the existing server.yaml and use controlSwitch to toggle the `injectHugepageDownApi` configuration for testing?

Reply:
+1 on this.

Reply:
I removed server_huge.yaml file from this change. But instead of using runtime feature, so controlSwitches, I stayed with the old solution. **injectHugepageDownApi** argument of webhook is added to the yaml by environment configuration script. I did it so, because I think that such approach requires less effort, and does not add additional dependency to test code.